Add "Open in codepen" buttons to 6 more examples

spectranaut opened the pull request:
Add "Open in Codepen" to 6 more examples!

Githack for review:
- [accordion/accordion.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/accordion/accordion.html)
- [grid/LayoutGrids.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/grid/LayoutGrids.html)
- [menu-button/menu-button-links.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/menu-button/menu-button-links.html)
- [grid/dataGrids.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/grid/dataGrids.html)
- [listbox/listbox-rearrangeable.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/listbox/listbox-rearrangeable.html)
- [listbox/listbox-collapsible.html](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/listbox/listbox-collapsible.html)

carmacleod reviewed (commented):
5 of 6 examples look good!

[datepicker dialog example](https://raw.githack.com/w3c/aria-practices/codepen-implementation-part6/examples/dialog-modal/datepicker-dialog.html) seems to be missing its CodePen button - maybe those changes weren't committed. :)

spectranaut commented:
whoops! I forgot to write when I fixed the comment, I accidentally linked to datepicker INSTEAD of layout grid (or was it data grid?) -- so I updated that link @carmacleod !

spectranaut commented on examples/grid/js/dataGrids.js:
```diff
@@ -12,27 +12,30 @@
-      toggleButton.innerText = 'Show Type and Category';
-    }
-  });
+  if (document.querySelector('#ex1-grid')) {
```
Yeah we should do that, I think it was a quick fix to check if it worked then I didn't go back and revisit the code. The reason is this: This code is set up to work expecting that all three examples are findable in the DOM -- and all three examples are initialized at once during this one section of javascript. But, we want to open all three examples separately in codepen -- so I just made these "ifs" to skip initializing the example if its not in the DOM, so each example CAN be opened separately in codepen.

spectranaut commented:
fixed the things you mentioned, take another look @carmacleod ? :)

mcking65 commented:
@carmacleod all good now?

carmacleod reviewed (approved):
@mcking65 
> @carmacleod all good now?

Yep, all good! Thanks again, @spectranaut !

mcking65 commented on q:
```diff
@@ -0,0 +1,74 @@
+[1mdiff --git a/examples/treegrid/js/treegrid-1.js b/examples/treegrid/js/treegrid-1.js[m
```
Not sure what this file is? Perhaps a working file accidentally included?

carmacleod replied:
Wow, sorry I missed that!
I see that it went in with the "Add Codepen button to data grids example" commit.
Since I had already reviewed the files for the 5 other examples, I jumped directly to the new data grids files to review them, so I missed seeing that sneaky extra file.
Nice catch, @mcking65 !

mcking65 reviewed (changes requested):
I think we have an accidental inclusion of some kind of temp working file.

spectranaut commented:
fixed! good catch @mcking65

spectranaut commented:
Resolved conflicts!

mcking65 reviewed (approved):
thank you @spectranaut!
